python/tidy/servo_tidy/tidy.py

```diff
@@ -295,9 +295,13 @@ def find_reverse_dependencies(dependency, version, content):
 def check_toml(file_name, lines):
     if not file_name.endswith(".toml"):
         raise StopIteration
+    mpl_licensed = False
     for idx, line in enumerate(lines):
         if line.find("*") != -1:
             yield (idx + 1, "found asterisk instead of minimum version number")
+        mpl_licensed |= ('license = "MPL-2.0"' in line)
+    if not mpl_licensed:
+        yield (0, ".toml file should contain MPL-2.0 license.")
```

Comment:
Oh good point. https://github.com/search?q=org%3Aservo+servo_tidy&ref=searchresults&type=Code&utf8=%E2%9C%93 shows `webrender_traits` as the only crate currently using `servo_tidy`. Is it worth future-proofing against non-MPL crates starting to use test-tidy?